src/app/container/App.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
```

Comment:
Vet ikke hvordan appen typisk har forholdt seg til `component` vs `container`, men tror vi beveger oss litt bort fra den vanlig oppfatningen om vi legger `App` i en `container` mappe.

Reply:
ja, sant, fant bare ikke på noen bra navn... har du et forslag? :-)

Reply:
Bruker som regel å gå for "wrapper"  de gangene jeg skriver "container" også kommer på at jeg ikke vil misbruke uttrykket. 😛